.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  vexchange-api-ci:
+    name: Vexchange API - CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: 📦 installing packages
+        run: npm install
+      - name: ⚙️ building...
+        run: npm run build
+      - name: 💅 code syntax and style
+        run: |
+          npm run format
```

Comment:
You don't want to be modifying source code after you've built + in a throwaway container. Just run lint only imo